web/src/components/ServiceList.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { Card, makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles(theme => ({
+    root: {
+        maxWidth: 400,
+        marginLeft: 10,
+        marginTop: 10,
+    },
+    title: {
+        fontSize: 14,
+    },
+    details: {
+        margin: '0 2px',
+    },
+    cardView: {
+        padding: theme.spacing(2),
+    },
+}));
+
+const ServiceList = () => {
+    const [services, setServices] = useState(null);
+    const classes = useStyles();
+    const fetchData = async () => {
+        const response = await axios.get('http://localhost:5000/services/');
+        setServices(response.data.result);
+    };
+    useEffect(() => {
+        fetchData();
+    }, []);
+
+    return (
+        <div>
+            <div className={classes.root}>
+                {services &&
```

Comment:
In javascript when the first value is falsey, it returns the actual value of the first argument when you try to short circuit like this.

So in this case, if services were null we would output "null" onto our page which we don't want to do. 
It would be preferred to switch this from `array && array.map(..)` to `array ? array.map(..) : null` because react will not render the null